tests/clients/test_broadcasts.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from berserk import Client
+from berserk.types.broadcast import BroadcastTopResponse
+from utils import skip_if_older_3_dot_10, validate
+
+
+@skip_if_older_3_dot_10
+def test_get_top_valid(requests_mock):
```

Comment:
This test does not test anything. There’s no logic in the code, so no need to test that. We actually want to check these server’s response, so it needs to be a VCR test, see CONTRIBUTING.